aeat.py

```diff
@@ -940,11 +940,13 @@ def get_issued_sii_reports(cls):
         issued_invoices['A1'] = periods2
 
         # search issued invoices [new]
+        company = Transaction().context.get('company', None)
```

Comment:
I'd suggest to simply set the _check_access flag in the wizard actions/transitions that end up here
```
def do_whatnot(self, action):
  with Transaction().set_context(_check_access=True):
    # whatever
    return action, {}
```
so that any other rule on any other model besides "User in company" can still apply so as to respect the former results.

A decorator could be developed for this purpose in the style of `trytond.model.modelstorage:without_check_access`.

Potentially, code from elsewhere could also lead to this point, and then one might expect quite the opposite behavior, e.g. if rules had been extended/overridden.